test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -989,12 +985,13 @@ test
     await t.expect(await enrollOktaVerifyPage.returnToAuthenticatorListLinkExists()).ok();
     await t.expect(await enrollOktaVerifyPage.signoutLinkExists()).ok();
 
-    // expect no polling for same device page
+    // expect polling for same device page
     if (!userVariables.gen3) {
+      await t.wait(4000);
```

Comment:
So we don't expect polling for gen3? It looks like we were originally gating `enrollSameDeviceMocks` w/ a request to `/challenge/poll` behind `gen3` variable above, though?

Reply:
Also, is the 4 sec `wait()` an arbitrary number or some standardized interval? Regardless, I'd suggest making this a constant at the top of the spec since I just took a peek and it seems like it's used at least 1 other time

Reply:
I also see `wait(30000)` used twice, if that is some sort of standardized interval it might make sense to make that a constant as part of this PR as well

Reply:
yeah gen 3 was actually always expecting polling while only gen 2 wasn't, thats why it was gated.  But if I recall correctly the polling behavior between gen 2 and gen 3 is different, with gen 3 resulting in different poll counts after 4 seconds thats why only gen 2 polling was being tested here.

Reply:
4 seconds is the standard interval between polls, so yeah good call ill make that a constant.  Yeah the wait(30000) is for the 'send again' reminder prompt ill make that a constant as well.